src/components/organisms/OrganismStatistic.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <div class="gallery">
+    <div class="mdc-card">
+      <div class="header">
+        <h1 class="mdc-typography--headline5">Statistik Lebensmittel</h1>
+      </div>
+      <div class="mdc-list-item__content">
+        <AtomDoughnutChart
+          class="graph"
+          :labels="labels"
+          :label="label"
+          :data="data"
+          :background-color="backgroundColor"
+          :hover-offset="hoverOffset"
+          :position="position"
+          :font-size="fontSize"
+        />
+      </div>
+    </div>
+    <div class="mdc-card">
+      <div class="header">
+        <h1 class="mdc-typography--headline5">Verfügbare Lebensmittel</h1>
+      </div>
+      <perfect-scrollbar
+        v-if="articles.length > 0"
+        tag="ul"
+        class="mdc-list"
+      >
+        <MoleculeArticleCard
+          v-for="article in articles"
+          :key="article.$id"
+          :article="article"
+          displayIcons="true"
+        />
+      </perfect-scrollbar>
+    </div>
+    <div class="mdc-card">
+      <div class="header">
+        <h1 class="mdc-typography--headline5">Konsumierte Lebensmittel</h1>
+      </div>
+      <perfect-scrollbar
+        v-if="articles.length > 0"
+        tag="ul"
+        class="mdc-list"
+      >
+        <MoleculeArticleCard
+          v-for="article in articles"
+          :key="article.$id"
+          :article="article"
+          displayIcons="true"
+        />
+      </perfect-scrollbar>
+    </div>
+    <div class="mdc-card">
+      <div class="header">
+        <h1 class="mdc-typography--headline5">Entsorgte Lebensmittel</h1>
+      </div>
+      <perfect-scrollbar
+        v-if="articles.length > 0"
+        tag="ul"
+        class="mdc-list"
+      >
+        <MoleculeArticleCard
+          v-for="article in articles"
+          :key="article.$id"
+          :article="article"
+          displayIcons="true"
+        />
+      </perfect-scrollbar>
+    </div>
+  </div>
+</template>
+<script lang="ts" setup>
+import { onMounted, PropType, ref } from 'vue';
+import MoleculeArticleCard from '../molecules/MoleculeArticleCard.vue';
+import AtomDoughnutChart from '../atoms/AtomDoughnutChart.vue';
+import { ArticleService } from '../../services/article.service';
+import { IExtendedArticle } from '../../interfaces/extended-article.interface';
+
+const articles = ref<IExtendedArticle[]>([]);
+
+defineProps({
+  labels:
+    {
```

Comment:
Remove line break on all Props
```suggestion
  labels: {
```